### ISSUE TITLE
Help Regarding UI

### ISSUE THREAD

**Issue by dhhruv** [dhhruv/YouTua#6]
Currently, I've been using Tkinter to make GUI for YouTua but it doesn't look the way people need. So, is there anyone who can configure YouTua with a better GUI library in Python. Also, if you think you can change the current GUI made using Tkinter to look better, it'll be useful. Please let me know if anything comes up in your mind.

**Comment by Sylack-lab** [comment]
Python doesn't excel in UI, that's a fact. There's various UI libraries, but I personally would recommend these:

- Flexx
- PyQT5
- PySide

**Comment by dhhruv** [comment]
Yeah @Sylack-lab , I did look into them but the thing is if I want to change, I'd have to do this from scratch plus I haven't used the libraries in any other projects so a helping hand would be good for the same.

**Comment by dhhruv** [comment]
> Hey @dhhruv !
> 
> Sorry for the late response, but I could come up only an exact clone of your Tkinter UI. I guess that the UI is already simple and compact, that said it makes really easy to navigate and get the job done!
> 
> Here are some screenshots:
> 
> <img alt="" width="300px" src="https://user-images.githubusercontent.com/47711025/139479129-1a955566-9895-4f05-b37c-e21c15461fc7.png">
> 
> <img alt="" width="300px" src="https://user-images.githubusercontent.com/47711025/139479152-8542954e-4bae-43de-9197-f18d47d31f4e.png">
> 
> <img alt="" width="300px" src="https://user-images.githubusercontent.com/47711025/139479453-464a60b6-7b47-4681-8cd2-37bd175b5f72.png">
> 
> But, I'll try to think of new designs.

That's good but if you can think to make it more attractive and more user-friendly, it would be better. Please let me know if you get any more ideas.

**Comment by nobalpha** [comment]
Hey @dhhruv !

Here's a small improvement to the UI. 

Can you tell me some other points to work on? Maybe new features? According to that, I can rebuild the UI with your suggestions.

<img src="https://user-images.githubusercontent.com/47711025/139587412-8b9e0b6b-5686-4703-b9f0-5322855c4723.png" width=300>
<img src="https://user-images.githubusercontent.com/47711025/139587436-856b3f07-d7aa-49f4-a14f-6c1c4762f7c6.png" width=300>
